### PR TITLE
feat : added auth page

### PR DESCRIPTION
**Pull request by ArjunJunna**
Have added authentication pages.

### CODE REVIEW

File: css/pages/auth.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  margin: 1rem 0;
+  position: relative;
+}
+
+.auth__container .form-input {
+  margin: 1rem 0;
+  padding: 1rem;
+  border: none;
+  outline: none;
+  width: 100%;
```
Comment (MayankKumar10) [inline css/pages/auth.css:30]: use responsive grid-view with 12 columns for width =100% so you don't have to repeat it

File: css/pages/auth.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+.auth__container .form-btn {
+  display: block;
+  margin: 2rem 0;
+  width: 100%;
+  padding: 0.8rem 1rem;
+  border-radius: 0.6rem;
+  border: none;
+  background-color: var(--primary-color);
```
Comment (MayankKumar10) [inline css/pages/auth.css]: use helper class for primary btn

File: css/pages/auth.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  display: block;
+  color: black;
+}
+
+.text-field span::before {
+  content: '';
+  position: absolute;
+  top: 40px;
+  left: 0;
+  width: 0%;
```
Comment (MayankKumar10) [inline css/pages/auth.css]: don't use % when use 0

File: css/pages/auth.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  position: relative;
+}
+
+.auth__container {
+  height: auto;
+  width: 35rem;
+  padding: 2rem;
+  background-color: #f5f3f3;
+  font-size: 1.2rem;
+  position: absolute;
```
Comment (MayankKumar10) [inline css/pages/auth.css]: make a helper class for absolute-container you can use this code another component as well

File: css/pages/auth.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+.auth__container .form-input {
+  margin: 1rem 0;
+  padding: 1rem;
+  border: none;
+  outline: none;
+  width: 100%;
+  border-bottom: 0.6px solid rgb(168, 164, 164);
+  background-color: #f5f3f3;
```
Comment (MayankKumar10) [inline css/pages/auth.css]: If you used this color elsewhere also define it in the root directory.

**Review by MayankKumar10 — APPROVED** [review]
Everything else looks good

File: css/pages/auth.css (hunk 1 above)
Comment (ArjunJunna) [reply css/pages/auth.css:30]: sure will try that. Thank you.

File: css/pages/auth.css (hunk 2 above)
Comment (ArjunJunna) [reply css/pages/auth.css]: This was really helpful one. Thank you.

File: css/pages/auth.css (hunk 3 above)
Comment (ArjunJunna) [reply css/pages/auth.css]: will fix that yes.

File: css/pages/auth.css (hunk 4 above)
Comment (ArjunJunna) [reply css/pages/auth.css]: ok will do that.

File: css/pages/auth.css (hunk 5 above)
Comment (ArjunJunna) [reply css/pages/auth.css]: sure will do that.

File: css/abstract/colors.css [hunk 6]
```diff
@@ -9,4 +9,8 @@
     --icon-color:grey;
```
Comment (jalajcodes) [inline css/abstract/colors.css]: you have `--grey-color` already you can use that.

**Review by jalajcodes — COMMENTED** [review]
After filling email when i click outside, it becomes like this
image

File: css/abstract/colors.css (hunk 6 above)
Comment (ArjunJunna) [reply css/abstract/colors.css]: made the changes thank you

**Pull request merged**